spec/controllers/admin/service_orders_controller_spec.rb

```diff
@@ -1,11 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe Admin::ServiceOrdersController, :type => :controller do
+  let(:user) { create(:admin_user) }
 
   describe "GET index" do
-    it "returns http found" do
+    it "responds successfully with an HTTP 200 status code" do
+      sign_in user
       get :index
-      expect(response).to have_http_status(:found)
+      expect(response).to be_success
+      expect(response).to have_http_status(200)
+    end
+
+    it "assigns @service" do
+      sign_in user
```

Comment:
line ``8, 9`` and ``15..18`` is the same, why don't move to ``before`` scope :kimono: 

Also should consider using ``context`` scope, cause 2 test cases have different expectations, right? :kimono: